### ISSUE TITLE
Add ESM export

### ISSUE THREAD

**Issue by decrek** [mdevils/css-selector-parser#22]
In order to use this package in an ESM only context the package needs to be compiled to ESM. Are you willing to support this?
If you want I can create a PR for this. If so, do you prefer ESM only or support both CommonJS and ESM?

**Comment by Siilwyn** [comment]
@nene maybe you have some time to look at this?

**Comment by nene** [comment]
@Siilwyn, sorry, but no. I've only contributed one small commit to this library :)

**Comment by mdevils** [comment]
Hello @decrek, this issue was fixed in version `v2.2.0`. Please update.

**Comment by mdevils** [comment]
Hello @decrek, ESM export was added in version `2.3.1`.

**Comment by noyobo** [comment]
@mdevils  ESM export need to declare an explicit file suffix, otherwise, it will report an error

```js
Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/node_modules/.pnpm/css-selector-parser@2.3.1/node_modules/css-selector-parser/dist/mjs/parser' imported from /node_modules/.pnpm/css-selector-parser@2.3.1/node_modules/css-selector-parser/dist/mjs/index.js
```

All imports should be modified as follows:

```diff
- export { createParser } from './parser';
+ export { createParser } from './parser.js';
export { render } from './render';
export { ast } from './ast';
```

**Comment by mdevils** [comment]
@noyobo Thanks for checking. Fixed in `2.3.2`.
